### DISCUSSION TITLE
Node label widths

### DISCUSSION THREAD

**Discussion by ndtreviv** [f5/unovis#111] [Q&A]
At the moment the node labels auto-ellipsis, which is great. But, I'm trying to create a key for my graph, and have done so by rendering another graph with sample data to show different node and link types. I'm using the node label to describe the node type and in this case I don't want it ellipsing.

Is it possible to expand the width of a node label to somehow avoid label shortening?

**Comment by rokotyan** [comment]
@ndtreviv At this moment it's not configurable, but we're planning add it within the next month or so.
